api/auth.js

```diff
@@ -8,116 +8,116 @@ const JWT_SECRET = process.env.JWT_SECRET;
 
```

Comment:
I did changes here but then reverted them cause bertha reminded me of the easier way. So everything is the same here.